### PR TITLE
stop building VS2010 OpenSSL

### PR DESCRIPTION
**Pull request by reaperhulk**
we don't need it now that we've dropped py27

### CODE REVIEW

File: .github/workflows/build-windows-openssl.yml [hunk 1]
```diff
@@ -24,21 +24,17 @@ jobs:
     strategy:
       matrix:
         ARCH: ["win32", "win64"]
-        MSVC_VERSION: ["2010", "2019"]
+        MSVC_VERSION: ["2019"]
```
Comment (alex) [inline .github/workflows/build-windows-openssl.yml:27]: Either we should remove this matrix axis, or we should leave the `if` below.

File: windows/openssl/build_openssl.bat [hunk 2]
```diff
@@ -3,17 +3,9 @@ SET VSVERSION=%2
 
 cd openssl-1*
 
-if "%VSVERSION%" == "2010" (
-    if "%BUILDARCH%" == "win32" (
-        CALL ..\windows\openssl\build_openssl_win32_2010.bat
-    ) else (
-        CALL ..\windows\openssl\build_openssl_win64_2010.bat
-    )
```
Comment (alex) [inline windows/openssl/build_openssl.bat:11]: Is there an assert or something we can use on `VSVERSION`

Comment (reaperhulk) [reply windows/openssl/build_openssl.bat:11]: I don't know if batch has an assert. We could leave the conditional and just wrap around the whole thing for now.

Comment (reaperhulk) [reply windows/openssl/build_openssl.bat:11]: Or we could stop passing VSVERSION to the script entirely

Comment (reaperhulk) [reply windows/openssl/build_openssl.bat:11]: (I removed VSVERSION entirely)

**Review by alex — APPROVED** [review]

**Pull request merged**